Sources/TSCExtensions/Process+Run.swift

```diff
@@ -36,7 +36,7 @@ import Android
 
 extension Process {
   /// Wait for the process to exit. If the task gets cancelled, during this time, send a `SIGINT` to the process.
-  /// Should the process not terminate on SIGINT after 2 seconds, it is killed using `SIGKILL`.
+  /// Should the process not terminate on SIGINT after 2 seconds, it is terminated using `SIGKILL`. # ignore-unacceptable-language
```

Comment:
No way to avoid this one but... The `# ignore-unacceptable-language` does show up in the help 🤔

Reply:
Ah, I believe the issue is only `kill` (the function) and not `SIGKILL`. Which makes sense, otherwise surely there'd be a fair few false positives with `kill` *anywhere* in a word.